platform/core/src/services/HangingProtocolService/themeProtocolProvider.js

```diff
@@ -0,0 +1,116 @@
+import ConfigPoint from 'config-point';
+
+const { ThemeProtocols } = ConfigPoint.register({
+  ThemeProtocols: {
+    configBase: {
+      protocols: [
+        {
+          id: '1x2',
+          locked: true,
+          hasUpdatedPriorsInformation: false,
+          name: '1x2',
+          createdDate: '2021-11-01T18:32:42.849Z',
+          modifiedDate: '2021-11-01T18:32:42.849Z',
+          availableTo: {},
+          editableBy: {},
+          protocolMatchingRules: [
+            {
+              id: 'NumberOfStudyRelatedSeries>1',
+              weight: 10,
+              attribute: 'NumberOfStudyRelatedSeries',
```

Comment:
Note that this SHOULD be NumberOfDisplaySets rather than number of study related series, as a series like a DICOM SR will get a box for it, but that shouldn't actually cause the rule to be activated.  But that is hard to arrange for because
the display set calculations aren't necessarily completed at this point.